### PR TITLE
Filter by dates implemented

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: src/main/java/com/fundation/logic/controller/ConvertController.java [hunk 1]
```diff
@@ -306,4 +310,37 @@ public void resetCriteria() {
...
         dpi = null;
     }
+
+    /**
+     * Listen to convert option on table result popupmenu.
+     */
+    public void listenConvertItem() {
+        try {
+            String path = mainFrame.getTableResult().getMenu().getFilePath();
+            System.out.println("path: " + path);
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/controller/ConvertController.java]: delete this print

File: src/main/java/com/fundation/logic/database/Query.java [hunk 2]
```diff
@@ -34,6 +34,7 @@ public void insertCriteria(String name, String json) {
         try {
             Timestamp current = Timestamp.valueOf(LocalDateTime.now());
             long dateLong = current.getTime();
+            System.out.println("date saved: " + dateLong);
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/database/Query.java]: delete this print

File: src/main/java/com/fundation/logic/controller/ConvertController.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/controller/ConvertController.java]: done

File: src/main/java/com/fundation/logic/database/Query.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/database/Query.java]: done

**Pull request merged**